readme.md

```diff
@@ -1,7 +1,15 @@
 # tokio-bin-process
 
+[![Crates.io](https://img.shields.io/crates/v/tokio-bin-process.svg)](https://crates.io/crates/tokio-bin-process)
+[![Docs](https://docs.rs/tokio-bin-process/badge.svg)](https://docs.rs/tokio-bin-process)
+[![dependency status](https://deps.rs/repo/github/shotover/tokio-bin-process/status.svg)](https://deps.rs/repo/github/shotover/tokio-bin-process)
+
 Allows your integration tests to run your application under a separate process and assert on tokio-tracing events.
 
+To achieve this it builds your application as a native binary,
```

Comment:
Not important, but I think there should be a comma? 🤔

> To achieve this, it builds...

(Ditto for repetition in comments below.)

Reply:
good point.
Thanks for the reviews btw!